pkg/raft/storage.go

```diff
@@ -40,15 +40,8 @@ var ErrUnavailable = errors.New("requested entry at index is unavailable")
 // snapshot is temporarily unavailable.
 var ErrSnapshotTemporarilyUnavailable = errors.New("snapshot is temporarily unavailable")
 
-// Storage is an interface that may be implemented by the application
-// to retrieve log entries from storage.
-//
-// If any Storage method returns an error, the raft instance will
-// become inoperable and refuse to participate in elections; the
-// application is responsible for cleanup and recovery in this case.
-type Storage interface {
-	// TODO(tbg): split this into two interfaces, LogStorage and StateStorage.
-
+// LogStorage is a read API for the raft log.
+type LogStorage interface {
 	// InitialState returns the saved HardState and ConfState information.
 	InitialState() (pb.HardState, pb.ConfState, error)
```

Comment:
I think this does belong to "log storage", but when we get to work on sep-raft-log again, some subset of it will also need to be read from the state machine storage to bootstrap the state upon startup (we would need to know who is ahead - log or the state machine).